packages/@react-spectrum/s2/stories/ComboBox.stories.tsx

```diff
@@ -46,7 +50,7 @@ type Story = StoryObj<typeof ComboBox<any>>;
 export const Example: Story = {
   render: (args: ComboBoxProps<any>) => (
     <ComboBox {...args}>
-      <ComboBoxItem>Chocolate</ComboBoxItem>
+      <ComboBoxItem id="1">Chocolate</ComboBoxItem>
```

Comment:
?

Reply:
oh lol, I think this was from a while back when I was testing something locally, thanks for catching